src/DbalSchemaCommand.php

```diff
@@ -23,15 +23,15 @@ public function __construct(Connection $db, SchemaDefinition $schemaDefinition)
     /**
      * Update the database schema to match the schema definition.
      */
-    public function update(bool $force, OutputInterface $output): void
+    public function update(bool $force, OutputInterface $output, bool $withTransaction = false): void
```

Comment:
The default behavior will change to not use transactions. Could we preserve the existing behavior?

Reply:
Yes, but the initial PR was not change the default behavior, but you proposed to switch the default behavior : https://github.com/mnapoli/dbal-schema/pull/12#discussion_r1053433402 Or want you just change behavior of options but not the method ?

Reply:
@frenchcomp OMG sorry 😅

I have so many prs and issues to track that I'm not remembering all the details, but thank you for clarifying! That sounds great then, I'll merge!